src/components/Memoroom.js

```diff
@@ -4,50 +4,156 @@ import styled from "styled-components";
 import PropTypes from "prop-types";
 import hashtag from "../assets/images/hashtag.png";
 import menu from "../assets/images/menu.png";
+import clickedMenu from "../assets/images/click-menu.png";
+import pen from "../assets/images/pen.png";
 import wastebasket from "../assets/images/wastebasket.png";
 
 const MemoRoomContainer = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
   align-items: center;
+  position: relative;
   width: 300px;
   height: 300px;
   border: 1px solid #000000;
+  background: #ffc300;
 
   img {
+    width: 40px;
+    margin: 15px;
+  }
+  .menu-icon {
     width: 20px;
     margin: 7px;
   }
 
-  .image-wrapper {
+  .menu-bar {
     display: flex;
     justify-content: flex-end;
+    position: absolute;
+    top: 5px;
+    right: 5px;
     width: 300px;
   }
 
+  .memoroom-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    height: 220px;
+    margin-top: 80px;
+  }
+
   .room-name {
-    margin-top: 100px;
     border: none;
     font-size: 25px;
   }
 
-  .tags {
+  .participant {
+    display: flex;
+    justify-content: center;
     width: 300px;
+    height: 50px;
     border-top: 1px solid black;
+    background: #fefbf2;
     font-size: 15px;
   }
+
+  .tags {
+    display: flex;
+    justify-content: center;
+    width: 200px;
+    height: 100px;
+    font-size: 15px;
+  }
+
+  .tag {
+    margin: 3px;
+    color: white;
+  }
+
+  .menu-click {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: absolute;
+    width: 300px;
+    height: 300px;
+    background: rgba(0, 0, 0, 0.6);
+  }
+
+  .menu-click-content {
+    margin-top: 100px;
+  }
 `;
 
 const MemoRoom = ({ id, roomName, tags }) => {
+  const [clickMenu, setClickMenu] = useState(false);
+  const [clickHashtag, setClickHashtag] = useState(false);
```

Comment:
```suggestion
  const [clickMemoRoomMenu, setMemoRoomClickMenu] = useState(false);
  const [clickMemoRoomHashtag, setMemoRoomClickHashtag] = useState(false);
```
이렇게 디테일하게 적어주시는 거는 어떠신가요?! 
이미 해당 컴포넌트 내부에 존재해서 이전에 작성해주신것도 좋은것 같은데 지수님, 수인님 의견 어떠신가요??

Reply:
너무 길어져서 넣지는 않았었는데 저렇게 하니 좀 더 직관적이어서 알아보기 쉬운것같아요! 저렇게 수정해도 좋을것 같습니다!

Reply:
넵 의견 감사합니다!!